app/lib/actions/pulp3/orchestration/orphan_cleanup/remove_orphans.rb

```diff
@@ -6,13 +6,14 @@ class RemoveOrphans < Pulp3::Abstract
           def plan(proxy)
             if proxy.pulp3_enabled?
               sequence do
-                plan_action(Actions::Pulp3::OrphanCleanup::DeleteOrphanRepositoryVersions, proxy)
                 if proxy.pulp_mirror?
                   plan_action(Actions::Pulp3::OrphanCleanup::RemoveUnneededRepos, proxy)
-                  plan_action(Actions::Pulp3::OrphanCleanup::DeleteOrphanDistributions, proxy)
                   plan_action(Actions::Pulp3::OrphanCleanup::DeleteOrphanAlternateContentSources, proxy)
                   plan_action(Actions::Pulp3::OrphanCleanup::DeleteOrphanRemotes, proxy)
                 end
+                # Deleting repos causes orphaned distributions, so delete them before the distributions.
+                plan_action(Actions::Pulp3::OrphanCleanup::DeleteOrphanDistributions, proxy)
+                plan_action(Actions::Pulp3::OrphanCleanup::DeleteOrphanRepositoryVersions, proxy)
```

Comment:
Can we add the rescued error(s) to the action[:output] for DeleteOrphanRepositoryVersions task so it's visible on the task directly along with the logs?

Reply:
I can try that, sure.

Reply:
![image](https://github.com/user-attachments/assets/2daceafa-eb20-456b-ab67-6f5789795ec2)

Reply:
Should pulp_tasks be hrefs of tasks?

Reply:
oops :)

Reply:
I think the true came from incorrectly collecting the tasks. I'll change it to directly grab the tasks.